deps/mbedtls.mk

```diff
@@ -49,7 +49,7 @@ $(MBEDTLS_OBJ_SOURCE): $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/Makefile
 
 $(BUILDDIR)/mbedtls-$(MBEDTLS_VER)/checked: $(MBEDTLS_OBJ_SOURCE)
 ifeq ($(OS),$(BUILD_OS))
-	$(MAKE) -C $(dir $@) test
+	-$(MAKE) -C $(dir $@) test
```

Comment:
so we should see the errors instead of skipping them if you manually run `make -C deps test-mbedtls`, no?